### PR TITLE
fix(suite): social metadata for suite-web and suite-web landing page

### PR DESCRIPTION
**Pull request by tomexx**
- fix typo in OG image filename
- fix Suite URL
- shared component for setting social sharing meta data for both suite-web and suite-web-landing

### CODE REVIEW

File: packages/suite-web-landing/pages/_document.tsx [hunk 1]
```diff
@@ -59,9 +59,7 @@ export default class MyDocument extends Document {
                     />
                     <meta
                         property="og:image"
-                        content={`${URLS.SUITE_URL}${resolveStaticPath(
```
Comment (slowbackspace) [inline packages/suite-web-landing/pages/_document.tsx:62]: you probably could still use resolveStaticPath just to be more future-proof, but should work anyway.

**Comment by slowbackspace** [comment]
I am wondering if these meta tags shouldn't also be used in suite-web package. What happens if someone will share suite.trezor.io/web (cause that's the route where suite is served). I think it won't work (and indeed looks like twitter can't handle it).

File: packages/suite-web-landing/pages/_document.tsx (hunk 1 above)
Comment (tomexx) [reply packages/suite-web-landing/pages/_document.tsx:62]: Probably yes, we can't test it properly on sldev anyway so at least the `static` path would't be hardcoded. I'll change it back

File: packages/suite/src/components/suite/Metadata/index.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { resolveStaticPath } from '@suite-utils/nextjs';
+import { URLS } from '@suite-constants';
+import Head from 'next/head';
+import { useTranslation } from '@suite-hooks/useTranslation';
```
Comment (slowbackspace) [inline packages/suite/src/components/suite/Metadata/index.tsx]: ahh, type-check is failing and I suspect this is the reason (useTranslation is importing something from suite-types, which means whole file will get type-checked and suite-web-landing pkg is missing some type definitions)

If I am correct then just don't use `useTranslation`, instead directly use `react-intl`'s `useIntl` hook

**Review by slowbackspace — APPROVED** [review]

**Pull request merged**